### PR TITLE
Use official docker image as base

### PR DESCRIPTION
**Pull request by krisgeus**

### CODE REVIEW

File: docker/airflow-python/Dockerfile [hunk 1]
```diff
@@ -1,59 +1,16 @@
...
 
-ENV AIRFLOW_BUILD_DEPS="freetds-dev python-dev libkrb5-dev libssl-dev libffi-dev libpq-dev git"
-ENV AIRFLOW_APT_DEPS="libsasl2-dev sasl2-bin libsasl2-2 libsasl2-dev libsasl2-modules freetds-bin build-essential default-libmysqlclient-dev apt-utils curl rsync netcat locales"
+USER root
 
-ENV AIRFLOW_VERSION=1.10.5
-ENV AIRFLOW_HOME /usr/local/airflow
-ENV AIRFLOW_GPL_UNIDECODE=yes
-ENV SLUGIFY_USES_TEXT_UNIDECODE=yes
+ARG AIRFLOW_VERSION=1.10.12
```
Comment (bjgbeelen) [inline docker/airflow-python/Dockerfile:7]: the `AIRFLOW_VERSION` arg was already defined in line 1.

File: envs/external-airflow-db/.whirl.env [hunk 2]
```diff
@@ -1,13 +1,14 @@
 # postgres env vars
 POSTGRES_HOST=postgresdb
 POSTGRES_PORT=5432
-POSTGRES_PASSWORD=p@ssw0rd
+POSTGRES_PASSWORD=pAssw0rd
```
Comment (bjgbeelen) [inline envs/external-airflow-db/.whirl.env:4]: Haha, was time to rotate that one ;-)

File: envs/postgres-s3-spark/.whirl.env [hunk 3]
```diff
@@ -13,9 +13,10 @@ POSTGRES_USER=postgres
 POSTGRES_DB=postgresdb
 
 # Spark variables
-SPARK_VERSION=2.4.1
+SPARK_VERSION=3.0.1
```
Comment (bjgbeelen) [inline envs/postgres-s3-spark/.whirl.env:16]: somewhere else you're referring to 3.0, perhaps we can be consistent and state the same version everywhere?

**Review by bjgbeelen — COMMENTED** [review]
Looks good! 
I saw some lines were commented out, perhaps these can be removed if no longer used to clean up a bit

File: envs/external-airflow-db/.whirl.env (hunk 2 above)
Comment (krisgeus) [reply envs/external-airflow-db/.whirl.env:4]: Ja en de extra `@` in de jdbcurl vorm zorgde ervoor dat de db connectie check niet goed ging.

File: docker/airflow-python/Dockerfile (hunk 1 above)
Comment (krisgeus) [reply docker/airflow-python/Dockerfile:7]: Ja maar voor zover ik weet zijn die na de FROM regel niet meer beschikbaar en dien je ze dus opnieuw te definieren.

File: envs/postgres-s3-spark/.whirl.env (hunk 3 above)
Comment (krisgeus) [reply envs/postgres-s3-spark/.whirl.env:16]: Yes. The other defined version relates to the tag of a docker image. There is no 3.0.1 tag unfortunately at the moment.

**Pull request merged**